longestString/longestString.js

```diff
@@ -1,4 +1,19 @@
 /*
  * Write a function that accepts an array of strings.
  * Return the longest string in the array.
- */
\ No newline at end of file
+ */
+
+const longestString = (arr) => {
+  let tempWord = 'notAWord';
+  let tempLength = 0;
```

Comment:
I love how you are keeping track of both the length of the string and the string itself. This keeps it modular and clean. You don't need to do this, but it is great for readability and reuse later!